src/components/Tile/Tile.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { MediaObject } from '../MediaObject';
+
+import Title from './components/Title';
+import TitleRow from './components/TitleRow';
+
+const Tile = (props) => {
+  const {
+    as,
+    board,
+    children,
+    className,
+    action,
+    figure,
+    title,
+    titleText,
+    ...rest
+  } = props;
+
+  const tileClasses = cx([
+    'slds-tile',
+    { 'slds-tile_board': board },
+    { 'slds-hint-parent': !!action },
+    className,
+  ]);
+
+  const tileContent = (
+    <React.Fragment>
+      <TitleRow action={action}>
+        <Title title={title} titleText={titleText} />
+      </TitleRow>
+      <div className="slds-tile__detail">
+        {children}
+      </div>
+    </React.Fragment>
+  );
+
+  if (figure) {
+    return (
+      <MediaObject
+        {...rest}
+        as={as}
+        center={false}
+        className={tileClasses}
+        figureLeft={figure}
+      >
+        {tileContent}
+      </MediaObject>
+    );
+  }
+
+  const El = as;
+  return (
+    <El {...rest} className={tileClasses}>
+      {tileContent}
+    </El>
+  );
+};
+
+Tile.defaultProps = {
+  as: 'article',
+  board: false,
+  className: null,
+  figure: null,
+  action: null,
+  titleText: '',
+};
+
+Tile.propTypes = {
+  /**
+   * HTML element name
+   */
+  as: PropTypes.string,
```

Comment:
not sure if we should impose limits here. We have a lot of places where we allow the string. Maybe we should go through all of the cases in `react-lds` in future and check where we should impose

Reply:
I just checked, don't think we have a problem here:
* `Card` (default `article`) most block-level elements would make sense here depending on content
* `Column,Grid` even more so, any block-level tag should be allowed and makes sense. For `Column`, inline tags are also ok
* `Container` -> any block level element